start.sh

```diff
@@ -74,8 +73,8 @@ fi
 
 # If these values aren't set or they're set to false
 # This syntax substitutes False if null or unset
-if [ "${IS_CRON_POD:-"false",,}" == "false" ]; then
-    if [ "${DEBUGPY_ENABLE:-"false",,}" == "false" ]; then
```

Comment:
What are those extra commas mean previously?

Reply:
I'm really not sure. I added them back last summer on 6f77ce7b93198e6aa53bd6112e9445a8c8dad8bd but I was trying to research them and couldn't figure out why I'd have done that.

If it's not set it sets the value to `"false",,` which doesn't compare. 

I [looked at this chart and this thread](https://stackoverflow.com/a/16753536/3708872) and it seems like either just having `-"false"` alone would do what I want which is what this code was like before this change.

So either I introduced a bug or there was some other reason for this that I'm can't find yet.

Reply:
I tested this this time with all 3 cases and put that in the issue for a test plan.
Testing set to false (is false)
Testing set to true (is true)
Testing not set at all (is false)

There is a problem if it's set to anything other than false/true (like no). It the comparison fails and it runs the cron pod else case.

I don't think this is a big deal as this is pretty explicit in the settings and true/false are non-string values in JSON.

Reply:
ok, I feel something is set other than true or false are don't have that option enabled at all the default should be Prod mode. But I am ok for now since i do see this property `"PTVSD_ENABLE": false,` in prod `env.json`

Reply:
Yeah, we could flip the logic here, but we do have it documented to false by default or unset so if someone does leave it with that configuration this does work. I feel like this should be completely redone in the future when we remove the cron stuff too as I feel like the internal cron didn't do much for us other than keep a log.